src/jwtManager.js

```diff
@@ -8,6 +9,22 @@ class JwtManager {
       return null;
     }
   }
+
+  async calculateAntiAbuseHash(props) {
```

Comment:
Make sure to take in the explicit properties by name in the expected order, validate there aren't any additional properties, before creating the valueString.